WordPress/Sniffs/WP/UseCapabilitiesNotRolesSniff.php

```diff
@@ -14,14 +14,30 @@
 /**
  * User capabilities should be used not roles.
  *
- * @link    https://make.wordpress.org/themes/handbook/review/required/#options-and-settings
- *
  * @package WPCS\WordPressCodingStandards
  *
  * @since   1.0.0
  */
 class UseCapabilitiesNotRolesSniff extends AbstractFunctionParameterSniff {
 
+	/**
+	 * Only check for known capabilites.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var boolean
+	 */
+	public $checkOnlyKnownCaps = true;
```

Comment:
Please use `$snake_case` property names here and below.

FYI: the WPCS native CS check does not check for this as often upstream properties are used/overruled, but (new) WPCS native properties should be snake_case.